backend/metasfresh-webui-api/src/main/java/de/metas/ui/web/view/ViewRestController.java

```diff
@@ -214,7 +229,12 @@ public JSONViewResult deleteStickyFilter(
 
 		final IView newView = viewsRepo.deleteStickyFilter(viewId, filterId);
 		final JSONOptions jsonOpts = newJSONOptions();
-		return JSONViewResult.of(ViewResult.ofView(newView), ViewRowOverridesHelper.getViewRowOverrides(newView), jsonOpts);
+		final ViewResult viewResult = ViewResult.ofView(newView);
+
+		final List<IViewRow> rows = viewResult.isPageLoaded() ? viewResult.getPage() : Collections.emptyList();
+		final IdentityHashMap<IViewRow, Boolean> documentsWithComments = commentsService.hasComments(rows);
+
+		return JSONViewResult.of(viewResult, ViewRowOverridesHelper.getViewRowOverrides(newView), jsonOpts,documentsWithComments);
```

Comment:
use our formatting... i.e. space after comma

Reply:
Forgot to indent.